people.md

```diff
@@ -0,0 +1,29 @@
+# pandas People
+
+## Benevolent Dictator for Life
+
+Wes McKinney is the Benevolent Dictator for Life (BDFL).
+
+## Project Core Team
+
+- Tom Augspurger
+- Phillip Cloud
+- Andy Hayden
+- Stephan Hoyer
+- Thomas Kluyver
```

Comment:
Ok, I will nix you from the list for now. thanks for the help in the past with Python3 support!